frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -155,7 +155,13 @@ const StoryCard = ({
           <div className="story-card-tags">
             {/* TODO: make tags that reflect actual state */}
             <Tag displayText={`Level ${level}`} />
-            {storyTranslationId && <Tag displayText={`${language}`} />}
+            {!storyTranslationId && storyId && (
+              <Tag
+                displayText={`${
+                  language[0] + language.substring(1).toLowerCase()
```

Comment:
I don't think that'll look correct for ASANTE_TWI,  ENGLISH_UK, ENGLISH_US, or ENGLISH_INDIA. Can you make a frontend util function for converting languages to title case? In that function, you can deal with the 4 edge cases but otherwise use this existing line of code. Feel free to make a new file in `frontend/src/utils`